pkg/filter/expr_filter.go

```diff
@@ -79,16 +77,10 @@ func newExprFilterRule(
 
 // verifyAndInitRule will verify and init the rule.
 // It should only be called in dmlExprFilter's verify method.
-func (r *dmlExprFilterRule) verify(tableInfos []*model.TableInfo, sqlMode string) error {
+func (r *dmlExprFilterRule) verify(tableInfos []*model.TableInfo) error {
```

Comment:
Please add some comments to explain that the expr must be written in default sql mode.